lib/src/tasks/coverage/api.dart

```diff
@@ -332,12 +332,8 @@ class CoverageTask extends Task {
     for (int i = 1; i < collections.length; i++) {
       if (!collections[i].existsSync()) continue;
       String coverage = collections[i].readAsStringSync();
-      if (coverage.isEmpty) {
-//        mergedJson['coverage'].addAll({});
-      } else {
-        Map coverageJson = JSON.decode(collections[i].readAsStringSync());
-        mergedJson['coverage'].addAll(coverageJson['coverage']);
-      }
+      Map coverageJson = JSON.decode(collections[i].readAsStringSync());
+      mergedJson['coverage'].addAll(coverageJson['coverage']);
```

Comment:
We still want the `isEmpty` check here to see if the file was completely empty. Something like this:

```dart
String coverage = collections[i].readAsStringSync();
if (coverage.isNotEmpty) {
  Map coverageJson = JSON.decode(coverage);
  mergedJson['coverage'].addAll(coverageJson['coverage']);
}
```